server/src/domain/asset/asset.service.ts

```diff
@@ -414,32 +416,71 @@ export class AssetService {
     const { ids, removeParent, dateTimeOriginal, latitude, longitude, ...options } = dto;
     await this.access.requirePermission(auth, Permission.ASSET_UPDATE, ids);
 
+    const stackIdsToCheckForDelete: string[] = [];
     if (removeParent) {
-      (options as Partial<AssetEntity>).stackParentId = null;
+      (options as Partial<AssetEntity>).stack = null;
```

Comment:
Can we type options from the beginning, so in l. 416?

Reply:
Probably, I mostly just kept things the same where I could